lib/rubocop/cop/rspec_rails/http_status.rb

```diff
@@ -64,6 +68,8 @@ class HttpStatus < ::RuboCop::Cop::RSpec::Base
         PATTERN
 
         def on_send(node)
+          return unless defined?(::Rack::Utils::SYMBOL_TO_STATUS_CODE)
```

Comment:
The performance of this line could probably be optimized, but given that it’s a temporary change I think it’s fine.

Reply:
@bquorning Good catch, 👍